validphys2/src/validphys/core.py

```diff
@@ -121,42 +116,71 @@ def label(self, label):
     @property
     def stats_class(self):
         """Return the stats calculator for this error type"""
-        error = self.ErrorType
+        error = self.error_type
         klass = STAT_TYPES[error]
-        if hasattr(self, 'ErrorConfLevel'):
-            klass = functools.partial(klass, rescale_factor=self.rescale_factor)
+        if self._has_error_conf_level():
+            klass = functools.partial(klass, rescale_factor=self._rescale_factor)
         return klass
 
-    #TODO: Make this a proper Path
     @property
     def infopath(self):
-        return lhaindex.infofilename(self.name)
+        return Path(lhaindex.infofilename(self.name))
+
+    @property
+    def info(self):
+        """Information contained in the LHAPDF .info file"""
+        if self._info is None:
+            try:
+                self._info = lhaindex.parse_info(self.name)
+            except IOError as e:
+                raise PDFDoesNotExist(self.name) from e
+        return self._info
+
+    @property
+    def q_min(self):
+        """Minimum Q as given by the LHAPDF .info file"""
+        return self.info["QMin"]
+
+    @property
+    def error_type(self):
+        """Error type as defined in the LHAPDF .info file"""
+        return self.info["ErrorType"]
+
+    def _has_error_conf_level(self):
+        """Utility method to check whether an info file contains ErrorConfLevel"""
+        return "ErrorConfLevel" in self.info
+
+    @property
+    def error_conf_level(self):
+        """Error confidence level as defined in the LHAPDF .info file
+        if no number is given in the LHAPDF .info file defaults to 68%
+        """
+        # Check possible misconfigured info file
+        if self.error_type == "replicas":
+            if self._has_error_conf_level():
+                raise ValueError(
+                    f"Attribute at {self.infopath} 'ErrorConfLevel' doesn't "
+                    "make sense with 'replicas' error type"
+                )
+            return None
+        return self.info.get("ErrorConfLevel", 68)
 
     @property
     def isinstalled(self):
         try:
-            self.infopath
+            return self.infopath.is_file()
```

Comment:
This looks a bit strange now. For one it is maybe raising `PDFDoesNotExist` elsewhere.

Reply:
How? It does not change with respect to what it was doing before, right?

Reply:
You are right.